### PR TITLE
Book Store: Styling

### PR DESCRIPTION
**Pull request by ryxtor**
## Hello! 👋🏻 

In this PR I added the style of the page to match [this project in Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) 🎨 

[Here are the requirements for this project.](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md) 📑 

[Here is the live demo.](https://ryxtor-bookstore.netlify.app/) 💻

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -21,48 +40,183 @@ h1 {
...
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
+  background-color: #fff;
+  cursor: pointer;
+}
```
Comment (micheaol) [inline src/App.css]: Please ensure that your final design should match Zeplin's design.

Your Implementation | Zeplin's design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ryxtor , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.css (hunk 1 above)
Comment (Whoistolu) [reply src/App.css]: All good now 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @ryxtor 

Great job implementing the recommended changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**